packages/dashmate/src/listr/tasks/updateNodeTaskFactory.js

```diff
@@ -0,0 +1,57 @@
+const _ = require('lodash');
```

Comment:
It's not using Listr so it must be outside of `listr/task`. Let's create `src/update` and name it `updateServiceImagesFactory`